akka-actor/src/main/scala/akka/dispatch/Mailbox.scala

```diff
@@ -391,7 +391,7 @@ class NodeMessageQueue extends AbstractNodeQueue[Envelope] with MessageQueue wit
  * Discards overflowing messages into DeadLetters.
  */
 class BoundedNodeMessageQueue(capacity: Int) extends AbstractBoundedNodeQueue[Envelope](capacity)
-  with MessageQueue with BoundedMessageQueueSemantics with MultipleConsumerSemantics {
+  with MessageQueue with BoundedMessageQueueSemantics {
```

Comment:
so `MultipleConsumerSemantics` was completely wrong?

@rkuhn what's your opinion about this change?